src/lib/run-dev.js

```diff
@@ -95,7 +95,7 @@ async function runDev (config, dataDir, options = {}, log = () => {}) {
         // note the condition: we still write backend urls EVEN if skipActions is set
         // the urls will always point to remotely deployed actions if skipActions is set
         log('injecting backend urls into frontend config')
-        urls = rtLibUtils.getActionUrls(devConfig, true, isLocal && !skipActions)
+        urls = rtLibUtils.getActionUrls(devConfig, true, isLocal && !skipActions, true)
```

Comment:
we should discuss when we are turning "legacy = false"

Reply:
The idea behind the legacy flag is to support the deprecated URL key.
By default `getActionUrls` will return the object with the new pk/action key as required, this alone will change the output of dist, and the output of `config.js`

So this flag will change the default output of the config.js, adding the support for the deprecated key.